src/features/surveys/components/surveyForm/SurveyFooter.tsx

```diff
@@ -0,0 +1,87 @@
+import { FC } from 'react';
+import { Box, List, ListItem, Typography } from '@mui/material';
+import Image from 'next/image';
+import Link from 'next/link';
+
+import messageIds from 'features/surveys/l10n/messageIds';
+import { Msg } from 'core/i18n';
+import SurveyContainer from './SurveyContainer';
+import theme from 'theme';
+
+const links = [
+  {
+    href: 'https://zetkin.org/',
+    text: messageIds.surveyFooter.links.foundation,
+  },
+  {
+    href: 'https://manual.zetkin.org/',
+    text: messageIds.surveyFooter.links.manual,
+  },
+  {
+    href: 'https://zetkin.org/privacy',
+    text: messageIds.surveyFooter.links.privacy,
+  },
+];
+
+const SurveyFooter: FC = () => {
```

Comment:
Maybe we can call this something more general, e.g. `PublicFooter` and then we can use it on all public pages?

Reply:
This sounds great. Where do you reckon is a good place for that in terms of the repo's directory structure? I'm perfectly happy to just do this as an in-place rename within `src/features/surveys/components/surveyForm` but it feels like there's a 50:50 chance you might have a clear home in mind for a general purpose component like this.

Reply:
Great question! Maybe we make it a `ZUI` component, e.g. in `src/zui/ZUIPublicFooter`?